test/runnable/test_cdvecfill.d

```diff
@@ -55,7 +55,7 @@ version (update)
         auto sink = appender!string();
         foreach (arch; [EnumMembers!Arch])
         {
-            auto args = [dmd, "-c", "-O", "-mcpu=" ~ arch.to!string, "test/runnable/test_cdvecfill.d"];
+            auto args = [dmd, "-c", "-fPIC", "-O", "-mcpu=" ~ arch.to!string, "test/runnable/test_cdvecfill.d"];
```

Comment:
This line in the update script isn't necessary itself because the default `dmd.conf` for Linux x86_64 already adds `-fPIC` as default flag, but imho explicitness helps.